sdk/maps/azure-maps-render/src/main/java/com/azure/maps/render/implementation/helpers/Utility.java

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.maps.render.implementation.helpers;
+
+import com.azure.maps.render.implementation.models.IncludeText;
+
+public class Utility {
+    public static IncludeText toIncludeTextPrivate(boolean includeText) {
+        return IncludeText.fromString(String.valueOf(includeText));
+    }
+}
```

Comment:
Is this utility class required? We can just inline this call to `fromString` method.